src/coffee/import.coffee

```diff
@@ -238,13 +238,18 @@ class Import
         else
           existingProduct.variants[variantInfo.index] = variant
         productsToUpdate[productIndex] =
-          product: existingProduct
+          product: _.extend existingProduct, @getProductLevelInfo _.deepClone entry.product
```

Comment:
This code has a problem when categories are set in sphere but not provided in import csv - they get replaced by an empty array - the same happens for other product-level attributes